apps/meteor/app/api/server/lib/users.ts

```diff
@@ -119,3 +119,95 @@ export function getNonEmptyQuery<T extends IUser>(query: Mongo.Query<T> | undefi
 
 	return { ...defaultQuery, ...query };
 }
+
+type findPaginatedUsersByStatusProps = {
```

Comment:
Small nitpick, don't we usually capitalize the first letter for types?

Reply:
You are correct, it was a little oversight on my side, will fix asap